README.md

```diff
@@ -63,7 +63,23 @@ You can watch industry experts talking about Volcano in different International
 
 The easiest way to deploy Volcano is to use the Helm chart.  Volcano can be deployed by cloning code and also by adding helm repo.
 
-## Using Volcano Helm Repo
+## Install Helm
+
+### Download Helm
+
+Install helm by following official guide - https://github.com/helm/helm/blob/master/docs/install.md
```

Comment:
I think it is too complicated, as a newer, he would not want to install  `A`, and then `B`, before actually install volcano, i would prefer moving this to a simple scripts.

No one like complicated thing. If possible, can we remove dependency on helm totally？ @k82cn @TommyLike

Reply:
@hzxuzhonghu maybe you can bring the practice from istio?

Reply:
I'm ok with that. IMO, it's complex because of security, e.g. RBAC; maybe we can provide a quick guidance which ignore security to make it simpler.

Reply:
FYI:
istio provides a generated istio-demo.yaml, which allows users to install easily(https://istio.io/docs/setup/kubernetes/install/kubernetes/).  And also provides helm install https://istio.io/docs/setup/kubernetes/install/helm/.

And when it comes to kubernetes, we can run ./hack/local-ip-cluster.sh to start a cluster for developing and testing.